public/extensions/miro.js

```diff
@@ -0,0 +1,98 @@
+import daily from "./core.js";
+import iframe from "./iframe.js";
+import callstate from "./callstate.js";
+
+let fp, iframeEl, call;
+
+let props = {};
+
+/* Extension configuration */
+// don't use the iframe extension's button
+iframe.configure({ url: "about:blank", buttons: {}, showUrl: false });
+
+/* Public interface */
+export default {
+  configure: function (p) {
+    Object.assign(props, p);
+  },
+};
+
+/* Private implementation */
+
+function iframeUrl(boardId) {
+  return "https://miro.com/app/live-embed/" + boardId;
+}
+
+function getTokenFromServer() {
+  return fetch("https://daily-miro.glitch.me/jwt");
+}
+
+function handleMiroButton() {
+  if (callstate.state.iframe?.open === true) {
+    callstate.updateCallState("iframe", {
+      open: false,
+    });
+  } else if (
+    callstate.state.iframe?.url &&
```

Comment:
If I remember correctly, this case needs to execute if `url` is defined inside `iframe` and set to something other than `about:blank`, but if `iframe` isn't defined at all (because of some quirks with the way call state works in this extension), it needs to fall down into the `else` block below